.golangci.yaml

```diff
@@ -12,7 +12,6 @@ linters:
     - staticcheck
     - dupl
     - errorlint
-    - exportloopref
```

Comment:
```
WARN The linter 'exportloopref' is deprecated (since v1.60.2) due to: Since Go1.22 (loopvar) this linter is no longer relevant. Replaced by copyloopvar.
```

Reply:
OK